readme.md

```diff
@@ -12,31 +12,74 @@
+3. Write down three examples of events.
+
+> If you need some help, you can find information on events and examples [W3Schools](http://www.w3schools.com/js/js_events.asp) and [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/Events).
+
+When you have finished, read [this issue](https://github.com/ga-wdi-lessons/js-events-callbacks/issues/9) and submit a comment with the three parts clearly labeled.
 
 ### User Interaction
 
-As we write client-side Javascript -- javascript that is executed by *our browsers* -- it is very important to keep the user's actions in mind when designing our app's UI.
+As we write client-side Javascript -- javascript that is executed in *the browser* -- it is very important to keep the user's actions in mind when designing our app's UI.
 
-For example, let's say we have a single button on our landing page, we need to write some code that will execute whenever a user clicks on that button.
+For example, let's say we have a single button on our landing page, we need to write some code that will run whenever a user clicks on that button.
 
 ### What is Asynchronicity?
 
-Javascript typically will run top-to-bottom. We as developers, however, have no idea when the code related to the button click will actually be executed. It's totally dependent on the user. Therefore, we need to write code that will execute asynchronously -- or in other words, outside of the typical top-to-bottom document flow -- and not hold up the rest of our application.
+In the JavaScript programs we have written so far, code runs in sequence; one line executing after another.
+When interpreting it, we start at the top of the file and read line by line to understand expected behavior.
+The flow of code goes from top to bottom.
+We can invoke a function which has been previously defined but we understand the code of the function body running line by line at the point of invocation.
 
-Javascript as a language was built with this problem in mind and has spawned many more solutions that have been introduced through libraries, packages and frameworks. Together, these tools provide some abstractions to interact with events. As such, we as developers can write code to listen for and respond to these events.
+In the case of our button however, when that code actually runs is totally dependent on the user.
+When a user clicks the button an event will be triggered.
+Therefore, we need to write code that will execute asynchronously -- kicking off when the user clicks.
+We do not want to hold up the rest of our application waiting for the user to click.
+
+Javascript as a language was built with this problem in mind and has spawned many more solutions that have been introduced through libraries, packages and frameworks as well as default language features.
+Together, these tools provide some abstractions to interact with events. As such, we as developers can write code to listen for and respond to these events.
```

Comment:
In this portion, either make sure to define what you mean by 'abstractions' or use a simpler word to ensure clarity (tools, ways, methods, etc.)